addons/arsenal/functions/fnc_fillLoadoutsList.sqf

```diff
@@ -50,10 +50,10 @@ if (GVAR(currentLoadoutsTab) != IDC_buttonSharedLoadouts) then {
             _loadoutCachedInfo = [_loadoutData] call FUNC(verifyLoadout);
             _contentPanelCtrl setVariable [_loadoutNameAndTab, _loadoutCachedInfo];
 
-            _loadoutCachedInfo params ["", "_nullItemsAmount", "_unavailableItemsAmount", "_nullItemsList", "_unavailableItemsList"];
+            _loadoutCachedInfo params ["", "_nullItemsList", "_unavailableItemsList"];
 
             // Log missing / nil items to RPT (only once per arsenal session)
-            if (GVAR(EnableRPTLog) && {(_nullItemsAmount > 0) || {_unavailableItemsAmount > 0}}) then {
+            if (GVAR(EnableRPTLog) && {(_nullItemsList isNotEqualTo []) || {_unavailableItemsList isNotEqualTo []}}) then {
```

Comment:
Doubt the lazy evals save anything here with checks this simple/quick.
```suggestion
            if (GVAR(EnableRPTLog) && (_nullItemsList isNotEqualTo [] || _unavailableItemsList isNotEqualTo [])) then {
```

Reply:
They do, lazy eval fail is still faster than creating an empty array.

Reply:
Interesting. Might be circumventable if pre-created, but, point.